pkg/nodeconfig/nodeconfig.go

```diff
@@ -193,13 +194,15 @@ func (nc *nodeConfig) Configure() error {
 			return errors.Wrap(err, "configuring WICD failed")
 		}
 		// Set the desired version annotation, communicating to WICD which Windows services configmap to use
-		if err := nc.applyLabelsAndAnnotations(nil, map[string]string{DesiredVersionAnnotation: version.Get()}); err != nil {
+		if err := metadata.ApplyLabelsAndAnnotations(nc.client, context.TODO(), *nc.node, nil,
```

Comment:
Adding a wrapper `metadata.ApplyVersionAnnotation()` will make the code readable when paired with `metadata.WaitForVersionAnnotation()`.